tests/test-rest-posts-controller.php

```diff
@@ -1441,6 +1520,19 @@ public function test_update_post_empty_content() {
 		$this->assertEquals( '', $new_data['content']['raw'] );
 	}
 
+	public function test_update_post_with_slashes_in_content() {
+		wp_set_current_user( $this->editor_id );
+
+		$request = new WP_REST_Request( 'PUT', sprintf( '/wp/v2/posts/%d', $this->post_id ) );
+		$request->set_body_params( array(
+			'content' => 'Hello\\s',
+		) );
+
+		$response = $this->server->dispatch( $request );
+		$new_data = $response->get_data();
+		$this->assertEquals( 'Hello\\s', $new_data['content']['raw'] );
+	}
```

Comment:
This test appears to be unrelated to the rest of the PR, can you split it into a new one? :)